src/Components/Common/Logo/Logo.module.scss

```diff
@@ -6,6 +6,10 @@
     font-size: 2.3rem;
     margin: 0 $indentMin 0 0;
 
+    span {
```

Comment:
If possible - let's refer to a class name instead of tag's name

Reply:
Done!